site/content/en/docs/tasks/run/deployment.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Run Deployment"
+linkTitle: "Deployment"
+date: 2024-07-25
+weight: 6
+description: >
+  Run a Deployment as a Kueue-managed job.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management
+capabilities when running Deployments.
+Although Kueue does not yet support managing a Deployment as a single workload, 
+it's still possible to leverage Kueue's scheduling and resource management capabilities for the individual Pods of the Deployment.
+
+In this section we demonstrate how to support scheduling Deployments in Kueue based on the Plain Pod integration,
```

Comment:
```suggestion
We demonstrate how to support scheduling Deployments in Kueue based on the Plain Pod integration,
```
since we already mention above "This page shows" setting the context.